build.gradle

```diff
@@ -1,5 +1,7 @@
 plugins {
     id 'java'
+    id 'jacoco'
+    id 'checkstyle'
 }
 
 group 'org.example'
```

Comment:
It would be nice to have here a bit more descriptive name :)

Reply:
Changed it:)